src/app/mycomments.js

```diff
@@ -140,8 +166,10 @@ class MyComments extends React.Component {
 
   renderComments(myComments, data) {
     const comments = []
+    console.log(myComments,data);
```

Comment:
can you please remove all the console.logs

Reply:
sure, they're for debugging but will do it